### DISCUSSION TITLE
Return To Origin When Complete

### DISCUSSION THREAD

**Discussion by LysolPionex** [arkypita/LaserGRBL#2075] [Ideas]
It would be nice to have an option to return the laser to its starting position.  I burn multiple things in succession, so if the laser could be put back where it starts each time, it'd save a lot of re-positioning time waste.

**Comment by arkypita** [comment]
Just edit footer code in settings.
Put G0 X0 Y0 

It was the default

**Reply by LysolPionex** [reply]
I could hug you right now.  I don't know why/how it wasn't in there, but this is so SO helpful!!  THank you so much!

**Reply by LysolPionex** [reply]
I just donated and found out you are in Italy!  We just came back from a trip there. :)
